inc/invitation.class.php

```diff
@@ -390,6 +404,10 @@ public function getSearchOptions() {
       return $tab;
    }
 
+   /**
+    * Deletes the invitation related to the entity
+    * @param CommonDBTM $item
```

Comment:
copy and adapt the comment from the same method in PluginFlyvemdmFile. I think it would be better.